docs/application/tizen-studio/rt-ide/overview.md

```diff
@@ -16,3 +16,5 @@ TizenRT is an RTOS-based lightweight platform. The following are the hardware an
 	TizenRT Studio version 2.0 (or later) is used as an IDE for developing IoT applications with st_things framework and ARTIK 053(s).
```

Comment:
Is this still valid?

Reply:
@manishtr can you please verify this. This must be removed since RT IDE is deprecated now.

Reply:
yes its still valid

Reply:
@manishtr thanks for verification.